tests/test_compute_service.py

```diff
@@ -168,6 +168,9 @@ def test_instance_properties(self):
             self.assertEqual(len(find_zone), 1,
                              "Instance's placement zone could not be "
                              " found in zones list")
+            self.assertIsNotNone(
+                test_instance.create_time,
+                "Instance must have it's creation time")
```

Comment:
```suggestion
                "Instance must have its creation time")
```

Reply:
Can we also add a test here to make sure that the returned object is always a python datetime object?